ledis/Support Files/AppDelegate.swift

```diff
@@ -17,32 +17,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         FirebaseApp.configure()
-        UINavigationBar.appearance().barTintColor = UIColor.blue
-        UINavigationBar.appearance().tintColor = UIColor.white
-        UINavigationBar.appearance().titleTextAttributes = [NSAttributedStringKey.foregroundColor : UIColor.white]
+        UINavigationBar.appearance().barTintColor = StyleManager.Colors.navigationBarBarTint
+        UINavigationBar.appearance().tintColor = StyleManager.Colors.navigationBarTint
+        UINavigationBar.appearance().titleTextAttributes = StyleManager.Colors.navigationBarTitleTextAttributes
```

Comment:
You can go further and configure this in StyleManager itself.
And you will only call single function here:
```swift
StyleManager.configureNavigationBarAppearance()
```